app/docs/md/learn/concepts/images/component.md

```diff
@@ -0,0 +1,154 @@
+---
+title: Enhance Image Component
+---
+
+Enhance Image is a [Single File Component (SFC)](/docs/learn/concepts/single-file-components) that makes authoring responsive images easy, by providing a simple custom element interface backed by an image transformation service.
+
+## Install
+
+Enhance Image comes preinstalled on all new Enhance projects. For preexisting Enhance projects, follow the directions below.
+
+Install Enhance Image via npm:
+
+```shell
+npm install @enhance/image
+```
+
+Then, add `enhance/arc-image-plugin` to your project’s `.arc` file, under the `@plugins` pragma:
+
+```arc
+@plugins
+enhance/arc-image-plugin
+```
+
+This will ensure that Enhance Image's image transformation service will be used in combination with the Enhance Image component.
+
+Finally, make the component available by exporting it from the package to your project:
+
+<doc-code filename="app/elements/enhance-image.mjs">
+
+```javascript
+export default EnhanceImage from '@enhance/image'
+```
+
+</doc-code>
+
+## Use
+
+```html
+<enhance-image
+  src='/_public/images/dog.jpg'
+  alt='My favourite dog'
+></enhance-image>
+```
```

Comment:
I probably missed this while travelling. According to the above it looks like we are going to add the plugin by default to the new projects. Are we going to add it to `arc-plugin-enhance` or `enhance-starter-project`. My vote would be `enhance-starter-project` as it would be easier for folks to remove `enhance-image` from their project if they decide they don't need it.

Reply:
My vote's also for the starter project yup! There may also be changes required to `arc-plugin-enhance` to enable passing the config into state but yeah, folks will be able to remove the plugin and component from the starter project easily.